### PR TITLE
tests: fix some machines flakes

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Comment by KKoukiou** [comment]
I can't reproduce this flake locally, tried normal run, run on loaded env, nothing makes it fail, though on CI it's quite consistent flake. Let's not merge this till we are sure that it helps with the flake.

**Comment by martinpitt** [comment]
Package build fails:
```
/tmp/source/pkg/machines/libvirt-dbus.js
  522:41  error  'returnProps' is not defined  no-undef
```

**Comment by martinpitt** [comment]
Heh, returnProps vs. resultProps :grin:

File: test/verify/machineslib.py [hunk 1]
```diff
@@ -824,6 +824,10 @@ def testVCPU(self):
         b.click("#vm-subVmTest1-run")
         b.wait_in_text("#vm-subVmTest1-state", "running")
 
+        # Wait for the VCPUs link to get new values before opening the dialog
+        b.wait_visible("#vm-subVmTest1-vcpus-count")
```
Comment (marusak) [inline test/verify/machineslib.py]: Heh, this is not gonna be needed when #11688 lands :)

Comment (KKoukiou) [reply test/verify/machineslib.py]: I will wait for that to land first then.

Comment (martinpitt) [reply test/verify/machineslib.py]: I'd rather land these flake fixes first, they (also) block #11688, and I think the latter might need more work. The extra wait_visible won't hurt.

**Comment by martinpitt** [comment]
[testVCPU failure on selenium/edge](https://209.132.184.41:8493/logs/pull-11749-20190506-124646-b9988e3a-selenium-edge/log.html#37), retrying that one. The other failures look like unrelated flakes.

**Comment by martinpitt** [comment]
All VM tests succeeded, no more testVCPU or "unknown property v" :fireworks:

**Pull request merged**